cdc/kv/etcd.go

```diff
@@ -60,6 +60,11 @@ const (
 	JobKeyPrefix = EtcdKeyBase + "/job"
 )
 
+const (
+	backoffBaseDelayInMs = 100
+	maxTries             = 3
```

Comment:
Is it better to make the variable name more describable, such as `defaultDDLMaxRetryTime` that we use. Ditto for the function `isRetryable`